### PR TITLE
set conn string to environment var if provided (solves issue#150)

### PR DESCRIPTION
**Pull request by VedantParanjape**
Crud is a map of Crudstub, you have commented that key to the map is the database type, but i dont see any place where it is stored, i.e. taking input from yaml/json file.

### CODE REVIEW

File: config/load.go [hunk 1]
```diff
@@ -3,11 +3,34 @@ package config
...
+	"os"
 	"strings"
 
 	"gopkg.in/yaml.v2"
 )
 
+func loadEnvironmentVariable(p *Project) {
+	if strings.HasPrefix(p.Secret, "$") {
+		tempString := strings.TrimPrefix(p.Secret, "$")
+		tempEnvVar, doesItExist := os.LookupEnv(tempString)
```
Comment (YourTechBud) [inline config/load.go]: Let's rename doesItExist to `p` or `present`?

File: config/load.go [hunk 2]
```diff
@@ -3,11 +3,34 @@ package config
...
+
+		if doesItExist {
+			p.Secret = tempEnvVar
+		}
+	}
+
+	for _, i := range p.Modules.Crud {
```
Comment (YourTechBud) [inline config/load.go]: Lets rename i to something like `c` (for crud) or `value`?

File: config/load.go [hunk 3]
```diff
@@ -3,11 +3,34 @@ package config
...
+		if strings.HasPrefix(i.Conn, "$") {
+			tempStringC := strings.TrimPrefix(i.Conn, "$")
+			tempEnvVarC, doesItExistC := os.LookupEnv(tempStringC)
+
+			if doesItExistC {
```
Comment (YourTechBud) [inline config/load.go]: Again doesItExist can be `p` or `present`

File: config/load.go (hunk 1 above)
Comment (VedantParanjape) [reply config/load.go]: okay, all done

File: config/load.go [hunk 4]
```diff
@@ -3,11 +3,34 @@ package config
...
+		}
+	}
+
+	for _, value := range p.Modules.Crud {
+		if strings.HasPrefix(value.Conn, "$") {
+			tempStringC := strings.TrimPrefix(value.Conn, "$")
+			tempEnvVarC, presentC := os.LookupEnv(tempStringC)
+
+			if presentC {
+				value = tempEnvVarC
```
Comment (YourTechBud) [inline config/load.go]:
```suggestion
				value.Conn = tempEnvVarC
```

**Pull request merged**